packages/vetrina/package.json

```diff
@@ -4,21 +4,22 @@
   "description": "Vetrina, frontend for purchasing KSF Media products",
   "license": "MIT",
   "scripts": {
-    "start": "parcel index.html -p 8000",
-    "build": "spago build && parcel build index.html --no-cache"
```

Comment:
No more `start` and `build` scripts? :thinking:

Reply:
I have no idea why this changes, need to revert them

Reply:
Ah no wait, I know why: I ran `yarn install` locally and it must have figured out that these scripts are unnecessary, because they were moved to the `package.json` of the `vetrina-staging` app. We now use that to produce local test builds.